cms/src/test/resources/test-data/api-specifications/ToPrettyJsonStringDeserializerTest/jsonObjectWithXmlValuePrettyPrinted.xml

```diff
@@ -0,0 +1,10 @@
+<MedicationRequest xmlns="http://hl7.org/fhir">
```

Comment:
is this correct? the file name implies this should be json containing XML, not a pure XML object